packages/sitecore-jss/src/personalize/graphql-personalize-service.ts

```diff
@@ -97,7 +99,7 @@ export class GraphQLPersonalizeService {
       return data?.layout?.item
         ? {
             // CDP expects content id format `<id>_<language>_<version>` (lowercase)
-            contentId: `${data.layout.item.id}_${language}_${data.layout.item.version}`.toLowerCase(),
+            contentId: `${EMBEDDED}${data.layout.item.id}_${language}`.toLowerCase(),
```

Comment:
Probably you can just write `embedded${data.layout.item.id}...` - anyway this prefix is not customizable, may be not needed to have a separate variable

Reply:
Yeah make sense. Thanks!